lib/wash/entry.rb

```diff
@@ -108,6 +108,27 @@ def description(description)
         @description = description
       end
 
+      # signals sets the entry's supported signals/signal-groups to signals. It is a helper for
+      # Entry schemas.
+      #
+      # @param signals An array of hashes where each hash represents a signal/signal group's schema.
+      #
+      # @example
+      #   class Foo
+      #     signals begin
+      #       [
+      #         # This is an example signal's schema
+      #         { name: "start", description: "Starts the entry" },
+      #
+      #         # This is an example signal group's schema. The 'regex' key distinguishes signal/signal group schemas.
```

Comment:
What's the format for this regex? Is it resolved in Ruby or Go?

Reply:
Thanks, updated.